codalab/rest/interpret.py

```diff
@@ -584,9 +584,12 @@ def expand_raw_item(raw_item):
         if is_search:
             keywords = rest_util.resolve_owner_in_keywords(keywords)
             bundle_uuids = local.model.search_bundle_uuids(request.user.user_id, keywords)
```

Comment:
Yikes this function is pretty confusing...perhaps call it `result` instead `bundle_uuids`.  In the future, we probably want to change this to return a dict with either `bundle_uuids` or `value`...